ui/pages/notifications-settings/notifications-settings-per-account.tsx

```diff
@@ -88,6 +71,8 @@ export const NotificationsSettingsPerAccount = ({
         address,
       },
     });
+    const fetchedData = await switchAccountNotifications([address]);
+    setData(fetchedData || {});
```

Comment:
NOICE

Reply:
Meh I'm a little worried we don't have error handling on async stuff, but we can do this feature wide in a future PR